packages/react-table/src/components/Table/SortColumn.tsx

```diff
@@ -30,7 +30,12 @@ export const SortColumn: React.FunctionComponent<SortColumnProps> = ({
     SortedByIcon = ArrowsAltVIcon;
   }
   return (
-    <button {...props} className={css(className, styles.tableButton)} onClick={event => onSort && onSort(event)}>
+    <button
+      type="button"
```

Comment:
If you want to make `type="button"` a default prop, do:
```
  sortDirection = '',
  type = 'button',
  ...props
}: SortColumnProps
```
...
`<button type={type}`
instead.

Reply:
Done